runtime/src/serde_snapshot.rs

```diff
@@ -232,6 +234,7 @@ pub(crate) fn compare_two_serialized_banks(
 /// Get snapshot storage lengths from accounts_db_fields
 pub(crate) fn snapshot_storage_lengths_from_fields(
```

Comment:
might be misremembering, but don't we process this part in serial? we can just grab the lock at the top of this fn instead of within every map.

Reply:
No more locking here.